utils.js

```diff
@@ -1,3 +1,3 @@
-export function makeStatsString(headCount, middleCount, bottomCount) {
-    return `You clicked on the head ${headCount} times, on the middle ${middleCount} times, and on the bottom ${bottomCount} time. And everybody loves your character's catchphrases:`;
+export function makeStatsString(skylineCount, castleCount, waterfrontCount) {
+    return `You clicked on the skyline ${skylineCount} times, on the castle ${castleCount} times, and on the waterfront ${waterfrontCount} times.`;
 }
```

Comment:
Remember to write a test for your `makeStatsString` function.  (This is the point you are missing on the deliverable)